tests/fixtures/plugins/testvendor/goto/Plugin.php

```diff
@@ -1,4 +1,4 @@
-<?php namespace TestVendor\Goto;
+<?php namespace TestVendor\_Goto;
```

Comment:
before php8 "goto" was a reserved word and was not allowed as part of the namespace. Now test checks for validity of plugin namespace according to PSR-4

Reply:
@tolkamsoft Unfortunately that's not what this fixture was being used for. It was to test that a [plugin with a parse error](https://github.com/octobercms/october/blob/develop/tests/unit/system/classes/PluginManagerTest.php#L52-L57) is not loaded. If having `goto` in the namespace is no longer a parse error, you would need to introduce a parse error somewhere else in this file.

Reply:
Actually I was referring to test plugin description:

> Test plugin used by unit tests to detect plugins with invalid namespaces

Reply:
Fixed now